deployment/template-service.yaml

```diff
@@ -168,4 +170,10 @@ parameters:
   displayName: Inference bucket name
   required: true
   name: AWS_BUCKET_NAME_INFERENCE
-  value: "avgupta-dev-gokube-triage"
\ No newline at end of file
+  value: "avgupta-dev-gokube-triage"
```

Comment:
That was only for testing purposes and I forgot to change it before pushing, changed it back to "latest" since we should be using master version everywhere once PR is merged. Fixed.